src/vs/platform/extensions/common/extensionHostStarter.ts

```diff
@@ -27,10 +26,9 @@ export interface IExtensionHostStarter {
 	onDynamicStdout(id: string): Event<string>;
 	onDynamicStderr(id: string): Event<string>;
 	onDynamicMessage(id: string): Event<any>;
-	onDynamicError(id: string): Event<{ error: SerializedError }>;
```

Comment:
@alexdima fyi the utility process does not provide a `utilityProcess.on("error")` equivalent as the node.js `process` does, so this went away.